app/views/compute_resources_vms/form/libvirt/_base.html.erb

```diff
@@ -36,3 +21,13 @@
                :help_block  => _("Image to use"),
                :label => _('Image')} %>
 </div>
+
+<script>
```

Comment:
could this be extracted into app/assets and included using `<%= javascript_include_tag '...libvirt...' %>`